static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -68,18 +69,34 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
   );
 }
 
-function getTitleSubtitleMessage(event: TimelineEvent) {
+// This function tries to imitate what getTitle() from utils.events does.
+// In that module, the data comes from the issues endpoint while in here
+// we grab the data from the events endpoint. A larger effort is
+// required in order to use that function directly since the data between
+// the two endpoint is slightly different.
+// For instance, the events endpoint could include a _metadata dict with
+// the title, subtitle and message.
+// We could also make another call to the issues endpoint  to fetch the metadata,
+// however, we currently don't support it and it is extremely slow
+export function getTitleSubtitleMessage(event: TimelineEvent) {
   let title;
```

Comment:
Since the try/catch was added, can we default this to something so that it can't be undefined if something goes wrong?